src/main/resources/application.yml

```diff
@@ -6,4 +6,12 @@ api:
     url: ${API_ERO_MANAGEMENT_URL}
 
 dynamodb:
-  print-details-table-name: ${DYNAMODB_PRINT_DETAILS_TABLE_NAME}
\ No newline at end of file
+  print-details-table-name: ${DYNAMODB_PRINT_DETAILS_TABLE_NAME}
+  scheduler-lock-table-name: ${DYNAMODB_SCHEDULER_LOCKS_TABLE_NAME}
+
+jobs:
+  print-requests:
+    name: "BatchPrintRequests"
+    cron: ${JOBS_PRINT_REQUEST_CRON}
+#    lock-at-most-for: ${JOBS_PRINT_REQUEST_LOCK_AT_MOST_FOR}
```

Comment:
Commented out?

Reply:
If/when you add these back in, I wonder if these are more suited to being values in here, rather than referenced as environment variables. EG:
```
jobs:
  print-requests:
    name: "BatchPrintRequests"
    cron: ${JOBS_PRINT_REQUEST_CRON}
    lock-at-most-for: PT5M
```
The thinking is, will this value (the 'lock at most for' for the print-request job) ever be different across environments? Will we have a different lock time for `dev` vs `int` vs `prod` for example?

If the answer is no/unlikely then we can do it without an environment variable, which makes infra etc a bit simpler.

(I would also suggest the same argument _might_ be true of the cron expression itself - will we want this triggering at different cron schedules in different environments? - can we just put the cron expression here and be done with it? Which would mean you dont need it in `application-test.yml` ?)

Reply:
I guess the lock expiry and the CRON are linked - if the lock expiry is longer than the CRON frequency then the CRON does not have the desired effect.  But I would suggest that is unlikely to happen in practice - I would guess the CRON frequency will not be less than 5 minutes and the expiry will not be more than 5 minutes.

For these I would assume setting sensible defaults in the application.yml will be sufficient - If required environment properties can always override (even if not defined explicitly here)

Reply:
Yes, agreed. I would imagine `dev` and `int` would be the only envs where we _might_ want something different - in examples like that perhaps the best approach would be to override the relevant properties in `application-dev.yml` etc
(we already do something similar in VCA to control feature flags in dev where we will never want a feature on)